airflow/settings.py

```diff
@@ -366,7 +366,15 @@ def import_local_settings():
         log.debug("Failed to import airflow_local_settings.", exc_info=True)
 
 
-def initialize():
+def preload_custom_imports():
+    """
+    Any custom imports that need to loaded and cached,
+    for example of custom hooks, sensors and operators
+    :rtype: None
+    """
```

Comment:
missing code?

Reply:
@feng-tao @astahlman It is not missing code. We can override it in airflowinfra, example -> https://github.com/lyft/airflowinfra/pull/445/files after this PR merged

Reply:
Isn't this invalid syntax in Python though? I thought you needed a `pass` statement

Reply:
yeah, surprise the CI won't fail as I thought that is invalid syntax

Reply:
Ok, apparently this is fine because of the docstring:

```
➜  ~ cat /tmp/without_docstring.py   # breaks
def hi():

➜  ~ python /tmp/without_docstring.py
  File "/tmp/without_docstring.py", line 3

    ^
SyntaxError: unexpected EOF while parsing
➜  ~ cat /tmp/with_docstring.py  # fine b/c it has a docstring
def hi():
    """A docstring"""
➜  ~ python /tmp/with_docstring.py
➜  ~ cat /tmp/without_docstring_with_pass.py  # fine b/c of the pass
def hi():
    pass
➜  ~ python /tmp/without_docstring_with_pass.py
➜  ~
```

Reply:
I'd recommended adding a `pass` just for clarity, and mentioning in the docstring that this is meant to be overridden

Reply:
Sure. Will do. I originally followed this example in airflow: https://github.com/lyft/incubator-airflow/blob/791a12c184e0033cbd7731c3673e7764bf33cba8/airflow/settings.py#L326-#L333